Modules/Sources/JetpackStats/Utilities/Modifiers/PulseAnimationModifier.swift

```diff
@@ -4,31 +4,28 @@ struct PulseAnimationModifier: ViewModifier {
     let isEnabled: Bool
 
     func body(content: Content) -> some View {
-        if isEnabled {
-            content
-                .mask {
-                    PulsingMask()
+        content
```

Comment:
Fixes an issue where this modifier was breaking the view identity of the `content` view by creating it in two separate branches, which lead to some animations issues.